packages/server/api/routes/products.router.js

```diff
@@ -41,6 +41,37 @@ router.get('/', (req, res, next) => {
       .catch(next);
   }
 });
+/**
+ * @swagger
+ * /products/search:
+ *  get:
+ *    tags:
+ *    - products
+ *    summary: Get all product Name Search
+ *    description:
```

Comment:
As the Sprint2 is stricter, I will ask for some naming changes as well 🙏🏽 

 *    summary: Search products by name
 *    Will return all products where the name matches the search input.
 *  etc.

Reply:
thank you for feedback. I will update.